.github/workflows/pipeline-segment-electron-test.yml

```diff
@@ -189,6 +229,36 @@ jobs:
             runuser -u builduser -- xvfb-run script/actions/run-tests.sh script/yarn test --runners=main --trace-uncaught --enable-logging --files $tests_files
           fi
         fi
+    - name: Setup enviroment variables for testing on Windows
```

Comment:
Do these "run tests" steps still need to be separate or was this for debugging, hoping it can be the same as the bash/unix block